tests/integration/workspace_access/test_redash.py

```diff
@@ -0,0 +1,88 @@
+import logging
+from datetime import timedelta
+
+from databricks.sdk.errors import NotFound
+from databricks.sdk.retries import retried
+from databricks.sdk.service import sql
+
+from databricks.labs.ucx.workspace_access import redash
+from databricks.labs.ucx.workspace_access.groups import GroupManager
+from databricks.labs.ucx.workspace_access.manager import PermissionManager
+from databricks.labs.ucx.workspace_access.redash import RedashPermissionsSupport
+
+logger = logging.getLogger(__name__)
+
+
+# Redash group permissions are cached for up to 10 mins.
+# therefore we need to allow at least 10 mins for checking the permissions after group rename.
+# On top, we have to allow some time to execute the rest of the test.
+@retried(on=[NotFound], timeout=timedelta(minutes=13))
+def test_redash_replace_workspace_groups_with_account_groups(
+    ws,
+    sql_backend,
+    inventory_schema,
+    make_ucx_group,
+    make_user,
+    make_query,
+    make_query_policy_permissions,
+):
+    """
+
+    Args:
+        sql_backend (object):
+    """
+    ws_group, _ = make_ucx_group()
```

Comment:
use `apply_tasks` and two workspace groups, the same way as other tests in this directory are doing.

Reply:
updated